src/cli_plugin/install/pack.js

```diff
@@ -70,6 +69,10 @@ async function mergePackageData(settings, packages) {
     pkg.version = _.get(packageInfo, 'version');
     pkg.name = _.get(packageInfo, 'name');
     pkg.path =  resolve(settings.pluginDir, pkg.name);
+    pkg.kibanaVersion = _.get(packageInfo, 'kibana.version');
+    if (!pkg.kibanaVersion) {
+      pkg.kibanaVersion = pkg.version;
```

Comment:
After re-reading the issue I wonder if it makes sense to make `kibana.version` optional? Why not just require it, and throw an error if it's missing? I think this will help guard against accidentally determining a plugin is compatible with Kibana, in situations where it just happens to be at the correct internal version number but the plugin developer is unaware whether it's actually compatible with that version of Kibana or not.